assets/swagger.json

```
@@ -4699,6 +4699,10 @@
       "type": "object",
       "title": "AppProjectSpec is the specification of an AppProject",
       "properties": {
+        "allDestinationsMustMatch": {
+          "type": "boolean",
+          "title": "AllDestinationsMustMatch determines whether a destination will short-circuit on the first match or if it will go through all destinations"
```

@
```suggestion
          "title": "AllDestinationsMustMatch determines whether all the destinations should be matched or skip the further matching if any one of the match completes"
```

I think above description needs refinement

---

How about 

```suggestion
          "title": "AllDestinationsMustMatch determines whether all destinations should be matched or to skip further matching if any one match completes"
```

?

---

both looks good to me

---

LGTM 😄